ui/src/components/HydraOperationForm.vue

```diff
@@ -68,12 +71,40 @@ export default defineComponent({
   },
   emits: ['submit', 'cancel'],
 
+  data (): { value?: GraphPointer | null; shapePointer?: GraphPointer } {
+    return {
+      value: null,
+      shapePointer: undefined
+    }
+  },
+
+  mounted () {
+    const { resource } = this
+
+    this.value = this.clone(resource)
+  },
+
+  watch: {
+    shape (shape) {
+      if (!this.shapePointer) {
+        this.shapePointer = shape?.pointer
+      }
+    },
+    resource (resource) {
+      this.value = this.clone(resource)
+    }
+  },
+
   computed: {
-    clone (): GraphPointer | null {
-      const { resource } = this
+    _submitLabel (): string {
+      return this.submitLabel ?? this.operation.title ?? 'Save'
+    },
+  },
 
+  methods: {
+    clone (resource: GraphPointer | undefined | null) {
```

Comment:
Question: What's the purpose of cloning the Clownface?

Reply:
God question. This is a consequence of what I think was bad design choice for Alcaeus. In the current version it keeps all dereferenced resources in a single dataset. This was meant to offer easy correlation of the resource representation and `ApiDocumentation` resource. Something I intend to change in a next major release.

The clone extracts only the actual resource triples to its own separate dataset